docs/spark-qualification-tool.md

```diff
@@ -493,6 +493,82 @@ details on limitations on UDFs and unsupported operators.
 11. _Exec Children Node Ids_
 12. _Exec Should Remove_: whether the Op is removed from the migrated plan.
 
+**Parsing Expressions within each Exec**
+
+The Qualification tool looks at the expressions in the _Exec_ to extract any duration information.
+Note that it is not possible to extract the expressions for each available _Execs_:
+- some Execs do not take any expressions, and 
+- some execs may not show the expressions in the eventlog. For example:
+
+  - `SortMergeJoin` shows only the columns
+    ```
+    df1.select(ceil(col("num")) as "a")
+       .join(df2.select(ceil(col("num")) as "b"), $"a" === $"b"))
+       .explain
+    
+    +- Exchange rangepartitioning(b#70L ASC NULLS FIRST, 200), ENSURE_REQUIREMENTS, [id=#221]
+    +- *(5) SortMergeJoin [a#68L], [b#70L], Inner
+    :- *(2) Sort [a#68L ASC NULLS FIRST], false, 0
+    :  +- Exchange hashpartitioning(a#68L, 200), ENSURE_REQUIREMENTS, [id=#204]
+    :     +- *(1) Project [CEIL(num#64) AS a#68L]
+    :        +- *(1) Filter isnotnull(CEIL(num#64))
+    :           +- *(1) ColumnarToRow
+    ```
+  - `CartesianProduct`
+    ```
+    joinDf1.crossJoin(joinDf2).explain
+    == Physical Plan ==
+    CartesianProduct
+    :- *(1) SerializeFromObject [input[0, int, false] AS value#37]
+    :  +- Scan[obj#36]
+    +- *(2) SerializeFromObject [input[0, int, false] AS value#43]
+    +- Scan[obj#42]
+    ```
+    
+The following table lists the exec's name, whether expressions show in the eventlog, and the status
+of expression parsing.
+
+| **  Exec **                             | **  Eventlog Expressions ** | **  Fully-Parsed ** |
+|-----------------------------------------|:---------------------------:|:-------------------:|
+|   AggregateInPandasExec                 |               -             |           -         |
```

Comment:
lets hold off on doing the documentation PR until we get more expressions done, we may want to add a column here stating we haven't implemented the parsing on it yet.